db/backups.go

```diff
@@ -0,0 +1,49 @@
+package db
+
+import (
+	"os"
+	"time"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	scalingo "github.com/Scalingo/go-scalingo"
+	humanize "github.com/dustin/go-humanize"
+	"github.com/olekukonko/tablewriter"
+	errgo "gopkg.in/errgo.v1"
+)
+
+func ListBackups(app, addon string) error {
+	client := config.ScalingoClient()
+	backups, err := client.BackupList(app, addon)
+	if err != nil {
+		return errgo.Notef(err, "fail to least backups")
+	}
+
+	t := tablewriter.NewWriter(os.Stdout)
+	t.SetHeader([]string{"ID", "Name", "CreatedAt", "Size", "Status"})
+
+	for _, backup := range backups {
+		t.Append([]string{
+			backup.ID,
+			backup.Name,
```

Comment:
Will the download backup function accepts the name of the backup as input or just the ID? If it does not accept the name, is it really worth displaying it? Form my perspective there is a risk it creates confusion into our user mind.

Reply:
Both

Reply:
Yeah you're right it might be confusing.